draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1068,6 +1108,44 @@ Disclosures:
 
 {{examples/w3c-vc/disclosures.md}}
 
+## Example 4b - W3C Verifiable Credentials Data Model v2.0, using JSON-LD
+
+This example illustrates how to use the artifacts defined in this specification to secure a payload
+that is represented as a W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0]
+and uses JSON-LD. The example uses a content type `credential+ld+json` defined in [@VC_DATA_v2.0], Section 6.3 in a `cty` JOSE Header value.
+
+SD-JWT is equivalent to an Issuer-signed W3C Verifiable Credential (W3C VC). Disclosures are sent alongside a VC.
+
+A Combined Format for Presentation with a Holder Binding JWT would be equivalent to a Holder-signed
+Verifiable Presentation as defined in [@VC_DATA_v2.0].
+
+In this example, Holder Binding is applied and a Combined Format for Presentation is signed
+using a raw public key passed in a `cnf` Claim in the SD-JWT.
```

Comment:
same comments, if these sections going to be identical, it would be nice to be DRY-er.

Reply:
```suggestion
using a Holder's public key passed in a `cnf` Claim in the SD-JWT.
```